lib/src/collection/multimap.dart

```diff
@@ -23,9 +23,10 @@ abstract class Multimap<K, V> {
   /// Constructs a new list-backed multimap.
   factory Multimap() => ListMultimap<K, V>();
 
-  /// Constructs a new list-backed multimap. For each element e of [iterable],
-  /// adds an association from [key](e) to [value](e). [key] and [value] each
-  /// default to the identity function.
+  /// Constructs a new list-backed multimap.
+  ///
+  /// For each element `e` of [iterable], adds an association from [key] to
+  /// [value]. [key] and [value] each default to the identity function.
```

Comment:
This is not what the constructor does. We're mapping from `key(e)` to `value(e)`.

Reply:
Updated, thanks!